hoomd/mpcd/ConfinedStreamingMethod.h

```diff
@@ -82,7 +82,8 @@ class PYBIND11_EXPORT ConfinedStreamingMethod : public mpcd::StreamingMethod
 
         //! Validate the system with the streaming geometry
         void validate();
-
+        //! GPUArray to mask the collision =1 if particle has collided with boundary
+        GPUArray<unsigned char> m_bounced;
```

Comment:
Please add this with the other data members above (after line 81). The documentation should be done as above too:
```suggestion
        GPUArray<unsigned char> m_bounced; //!< Flag for if particles were bounced from boundary
```